honeybee_radiance/cli/translate.py

```diff
@@ -0,0 +1,197 @@
+"""honeybee radiance translation commands."""
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        'click is not installed. Try `pip install . [cli]` command.'
```

Comment:
I think in most cases they actually need to download it from PyPI.

```suggestion
        'click is not installed. Try `pip install honeybee-radiance[cli]` command.'
```